task-2/src/components/Button/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+import './index.css'
+
+export const Button = () => {
+    return (
+        <>
+           
```

Comment:
Consider avoiding leaving empty/redundant components.

Reply:
All the content from unused components files have been removed.